packages/design-system-scripts/yarn.lock

```diff
@@ -6218,7 +6218,7 @@ inherits@2.0.3:
   resolved "https://registry.yarnpkg.com/inherits/-/inherits-2.0.3.tgz#633c2c83e3da42a502f52466022480f4208261de"
   integrity sha1-Yzwsg+PaQqUC9SRmAiSA9CCCYd4=
 
-ini@1.3.8, ini@^1.3.4:
+ini@^1.3.4:
```

Comment:
@bernardwang @line47 - what would be the best way to handle this pr as it will end up reversing the recent [Bump ini from 1.3.5 to 1.3.8](https://github.com/CMSgov/design-system/pull/905).

Reply:
I wonder if we ignore this and close the PR?

Reply:
Ah, i just checked the change for Bump ini from 1.3.5 to 1.3.8. and looks like it just updated the version for 'init' - maybe I can just keep those changes.

Reply:
This PR doesnt revert `Bump ini from 1.3.5 to 1.3.8`, it includes other lock file changes as well. I don't really see an issue with testing and merging this to get rid of the security alert. `.lock` files are used to deterministically install dependencies, so it makes sense to update them to fix security issues